### DISCUSSION TITLE
No network connection when a systemwide tunnel attached via TUN/TAP is active

### DISCUSSION THREAD

**Discussion by nphuracm** [waydroid/waydroid#1287] [Q&A]
I use Cloudflare WARP on my computer in `tunnel_only` mode - this creates a TUN/TAP interface named `CloudflareWARP` and denies access to the interface that directly points to the physical NIC, in my case `enp4s0`.
When WARP is inactive, the container's UI status bar shows an Ethernet icon. But with WARP active, the icon is gone. The `waydroid0` and `veth*@if2` interface are still there though, at least as shown in my host shell. The container's ethernet interface is unattached when running `ip addr show` in the container shell.
I do have a rather good reason to keep WARP active at all times. I am currently looking for a way to attach Waydroid container's ethernet to this TUN/TAP interface instead.

**Comment by ruki-qq** [comment]
Same problem with nekobox tun, any solution?
